### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by bilaliftikhar-arbisoft**
https://www.loom.com/share/a6ab1759d67641839bf003a2feb4fca6

### CODE REVIEW

File: src/components/login.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { useEffect } from "react"
+import { logout } from "../services/auth"
+
+const Login = () => {
+  useEffect(() => {
+    logout(() => { });
```
Comment (Taimoormk) [inline src/components/login.js]: Why are you passing an empty function to `logout` function?

File: src/components/navigation/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { isLoggedIn } from "../../services/auth"
+
+export { NavPrivate } from "./private"
+export { NavPublic } from "./public"
+
+export const NavBar = () => {
+  return (
+    <div
+      style={{
```
Comment (Taimoormk) [inline src/components/navigation/index.js:12]: For now style in jsx is ok, but consider to move it to styles file.

File: src/components/navigation/private/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React, { Fragment } from "react"
+import { Link } from "@reach/router";
+
+export const NavPrivate = () =>
+  <Fragment>
+    <Link to='/app/profile'>Profile</Link>
```
Comment (Taimoormk) [inline src/components/navigation/private/index.js]: Write a an iterator function that will build this jsx when provided with an array of objects with all the desired routes config. Make sure to add `enabled: true` property to switch any route off/on for better usage.

**Review by Taimoormk — COMMENTED** [review]
@bilaliftikhar-arbisoft Good attempt, there are a few comments for your attention.

**Review by Taimoormk — APPROVED** [review]
@bilaliftikhar-arbisoft Looks good, once you worked on the comment, please mark as resolved too.

File: src/components/navigation/index.js (hunk 2 above)
Comment (bilaliftikhar-arbisoft) [reply src/components/navigation/index.js:12]: okay

**Pull request merged**